src/content/en/tools/chrome-devtools/beginners/devtools-for-beginners1.md

```diff
@@ -0,0 +1,239 @@
+
+ <figure> <img src="elpanel.png" alt=" The Elements Panel Opened" width="auto" height="auto">
+<figcaption>
+    <b>Figure 4</b>. The elements panel.
+  </figcaption>
+ </figure>
+
+2. Right click on the `<About>` div, then click `Edit as HTML`. 
+The following screen will appear:
+
+<figure> <img src="editashtml.png" alt="Editing as HTML" width="auto" height="auto">
+<figcaption>
+    <b>Figure 5</b>. Editing an object as HTML.
+  </figcaption>
+</figure>
+
+Now you have a live view of your changes! 
+
+Try adding information to the About node:
+
+1. Add a paragraph element like: 
+`<p> My website is all about sharing my  achievements! </p>`. 
+You can do this by clicking under `<div class= “About”>` 
+and typing in your code.
+2. Add a header element like `<h4>Here’s my resume.</h4>` 
+under the paragraph element.
+3. Add a button under the header element with 
+`<button>Download it!</button>`
+4. Copy your code and paste it under line 13 inside the “About” `<div>`.
+
+Note: If you refresh the page or close the tab, 
+your edits will be gone forever. After all, 
+imagine what would happen if you could permanently change 
+the HTML and text of any website! So,
+make sure to copy any changes you make to your site in DevTools.
+
+If you know what you want to edit, then there’s an even simpler workflow:
+
+1. Open the viewing tab.
+2. Highlight a section of your website (like an image or line of text).
+3. Right click and choose **Inspect**. 
+4. Double click on either the content or tag of the element to edit it.
+
+<figure> <img src="inspectel.png" alt="Inspect Element" width="auto" height="auto">
+<figcaption>
+    <b>Figure 6</b>. Clicking Inspect Element.
+  </figcaption>
+</figure>
+
+You can inspect any element using this method on any web page, 
+but note that like before, 
+your changes are not automatically saved when you edit through DevTools.
+
+## Reordering the DOM Tree {: #dom-tree}
+
+Another workflow for editing the arrangement 
+of your site is to rearrange the 
+DOM Tree of your document. For example, on your site, 
+you'll see that the navigation menu is on the bottom of the site. 
+To move it to the top:
+
+1. If you aren't already in the Elements Panel in DevTools,
+navigate to it.
+2. Scroll down the DOM Tree until you reach the `<nav>` `<div>`.
+3. Click and hold down on the `<nav>` `<div>`, 
+then drag it to the area you want it to be in. 
+
+**You’ve successfully learned how to add and edit content 
+on your site with DevTools!** If you want more information on these 
+workflows, take a look at [Inspect Styles](src/content/en/tools/chrome-devtools/inspect-styles/edit-styles.md)
+.
+
+
+## Next Steps {: #next-steps}
+<figure> <img src="endgame.png" alt="Prepare for the next steps" width="auto" height="auto">
+<figcaption>
+    <b>Figure 7</b>. An example of what the finished HTML of your site might look like.
+  </figcaption>
+</figure>
+
+Now that you have a grasp on how DevTools interacts with HTML, you’ll be
```

Comment:
This may need to change, so that you aren't dependent on the CSS tutorial for publication.